src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -0,0 +1,13 @@
+package mate.academy.service;
+
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
+import mate.academy.model.User;
+
+public interface AuthenticationService {
+
+    User login(String email, String password) throws AuthenticationException;
+
+    User register(String email, String password) throws RegistrationException;
+

```

Comment:
![image](https://github.com/mate-academy/hibernate-user-service-hw/assets/156338484/b3c1438d-062c-4cc0-8803-6e7a3e7d7e71)
Keep the same style